src/doc/trpl/traits.md

```diff
@@ -19,7 +19,8 @@ impl Circle {
 
 [methodsyntax]: method-syntax.html
 
-Traits are similar, except that we define a trait with just the method
+[Traits](../reference.html#traits) are similar,
```

Comment:
Alternatively, how/where would you insert the definition of *trait* in this chapter?